django/thunderstore/core/settings.py

```diff
@@ -521,7 +521,10 @@ def show_debug_toolbar(request: HttpRequest) -> bool:
 
 DEFAULT_FILE_STORAGE = "django.core.files.storage.FileSystemStorage"
 THUMBNAIL_DEFAULT_STORAGE = "django.core.files.storage.FileSystemStorage"
-PACKAGE_FILE_STORAGE = "django.core.files.storage.FileSystemStorage"
+if env.bool("DEBUG"):
```

Comment:
On further thought, instead of simply checking `manage.py` was executed, it'd be best to whitelist specific subcommands that can be used with invalid storage configuration. In this case I'd say those subcommands should be `makemigrations`, `collectstatic`, and `migrate`. Otherwise it would be possible to have a custom management command that relies on the storage be run w/o proper storage configuration.

Feel free to unify the stub-storage check into a single function e.g. by extending the `is_makemigrations_run` in `thunderstore.utils`, something like `should_use_stub_storage` which contains all the clauses & is used consistently everywhere.